setup.py

```diff
@@ -28,7 +28,7 @@
     # used to import the logging config file into pydarn.
     data_files=[('pydarn',['pydarn/logging_config.yaml'])],
```

Comment:
Although not really a change in this PR, I couldn't get pydarn to import without changing this line to:

    data_files=[('pydarn',['logging_config.yaml'])],

Apparently, data_files isn't the best thing to be using? This guy even says using package_data is bad and that one should use a MANIFEST.in file instead: https://github.com/pypa/sampleproject/issues/30#issuecomment-143947944

Reply:
This is within the scope of the PR since it is a problem for someone to install. I will look at this option and a couple others I saw a while back.